app/components/application/MemberRoleCard/MemberRoleCard.tsx

```diff
@@ -0,0 +1,25 @@
+import Card from "../../shared/Card"
+import Text from "../../shared/Text"
+import styles from "./styles.css"
+
+/* c8 ignore next */
+export const links = () => {
+  return [{ rel: "stylesheet", href: styles }]
+}
+
+type MemberRoleProps = {
+  role: "Owner" | "Member"
```

Comment:
Everything looks good, but I think we're missing the Admin role, right?

Reply:
did you see Admin somewhere else? from what I recall there was only Owner and Member but I may be wrong